### PR TITLE
Add from imdb watchlist

### PR DESCRIPTION
**Pull request by p0psicles**
First version released for public testing of the imdb watchlist feature.

It works as follows. The page Shows -> Add Shows -> Add from imdb popular shows, isn't called that anymore. The popular shows list, is 'à' list that you can use from imdb. We've added the feature to add your own watchlists, which will become visible once added and enabled in Config -> General Configuration -> Watchlists.

You only need to open imdb.com, login and navigate to your watchlist. I've all went well the url will look something like this: http://akas.imdb.com/user/ur59683423/watchlist?ref_=wt_nv_wl_all_0.

The most important thing here is that the list is public and that the url has the ur459858394 in it.
We still need to handle allot of exceptions. But thats why this is stil WIP.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [ ] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
Its in a feature branch, no reason not to merge it.

**Comment by labrys** [comment]
lol probably should have looked at that

File: gui/slick/views/addShows_imdbShows.mako [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<%inherit file="/layouts/main.mako"/>
+<%!
+    from sickbeard.helpers import anon_url
+    import sickbeard
+%>
+<%block name="metas">
+<meta data-var="sickbeard.SORT_ARTICLE" data-content="${sickbeard.SORT_ARTICLE}">
```
Comment (OmgImAlexis) [inline gui/slick/views/addShows_imdbShows.mako:7]: Don't we already add this var in the base file?

File: gui/slick/views/config_general.mako [hunk 2]
```diff
@@ -13,6 +13,7 @@
 %>
 <%block name="scripts">
 <script type="text/javascript" src="${srRoot}/js/rootDirs.js?${sbPID}"></script>
+<!-- <script type="text/javascript" src="${srRoot}/js/imdbWatchlists.js?v=${sbPID}"></script> -->
```
Comment (OmgImAlexis) [inline gui/slick/views/config_general.mako:16]: Remember to remove this as we only use `core.js`.

**Comment by p0psicles** [comment]
Yes your right will do. But there will probably be some more cleanup to do.

**Comment by bitzorro** [comment]
whats happening whit this project, has it stopped?

regards bitzorro

**Comment by p0psicles** [comment]
It never got through QA. Also i moved to another fork.

**Comment by bitzorro** [comment]
Link to New fork?

**Comment by p0psicles** [comment]
Sorry cant do that

**Comment by miigotu** [comment]
No, it hasn't stopped, we just slowed a bit and are more careful with merges, and releases are closer together ever two weeks rather than every week. Stability ensues.

**Comment by jaffacake** [comment]
Any update on this? Would be a really nice feature

**Comment by sharkykh** [comment]
Sorry, got a bit confused. ~@jaffacake This has been available for quite some time. In SR: **Shows > Add Shows** there's an IMDb option there.~

**Comment by jaffacake** [comment]
If there was a link or access to the branch/fork this is on I'd be happy to contribute...

**Comment by miigotu** [comment]
You could just add it here, we don't promote competing forks here.
Currently there is only the option to add from trakt watch lists.

**Pull request merged**